app.json

```diff
@@ -38,14 +59,13 @@
         "origin": false
       },
       "eas": {
-        "projectId": "ca2a4644-0c86-4092-b1f9-d554abfc52cf"
+        "projectId": "3196a892-2eac-48b2-89b1-69cb37e04986"
       }
     },
-    "runtimeVersion": {
-      "policy": "appVersion"
-    },
+    "runtimeVersion": "1.0.0",
     "updates": {
-      "url": "https://u.expo.dev/ca2a4644-0c86-4092-b1f9-d554abfc52cf"
-    }
+      "url": "https://u.expo.dev/3196a892-2eac-48b2-89b1-69cb37e04986"
+    },
+    "owner": "lyton505"
```

Comment:
I will remove this line. In the next meeting I will ask Lewis about making a client eas account.